apps/openmw/mwmechanics/stat.cpp

```diff
@@ -1,4 +1,5 @@
 #include "stat.hpp"
+#include "limits.h"
```

Comment:
OK, fixed. I also changed it to the minimum, rather than the maximum, since it seems very slightly less likely to be used. Probably it should be stored in the saved game file, though, since this PR is already a little bit problematic with existing save games, as I mentioned above. But I'm cautious about adding to the save data.